src/rt/dmain2.d

```diff
@@ -558,6 +558,35 @@ private void formatThrowable(Throwable t, scope void delegate(in char[] s) nothr
     }
 }
 
+private auto parseExceptionOptions()
+{
+    static struct ExceptionOptions
+    {
+        bool trap;
+
+        @nogc nothrow void initialize()
+        {
+            import rt.config : rt_configOption;
+            trap = rt_trapExceptions;
+            auto res = rt_configOption("extrap");
+            switch(res)
+            {
+                case "1", "y", "Y", "on", "true":
+                    trap = true;
+                    break;
+                case "0", "n", "N", "off", "false":
+                    trap = true;
+                    break;
+                default:
+                    break;
+            }
+        }
+    }
+    ExceptionOptions options;
```

Comment:
Still looks a little verbose to use a struct for a single bool.

Reply:
Yeah, see prior work: https://github.com/dlang/druntime/blob/8c86106a56f752455d387ce9d835b95fe7192e12/src/rt/lifetime.d#L48

Also, you have both setting `trap` to `true`!